genesyscloud/task_management_worktype_status_transition/resource_genesyscloud_task_management_worktype_status_transition_schema.go

```diff
@@ -0,0 +1,119 @@
+		ReadContext:   provider.ReadWithPooledClient(readTaskManagementWorkTypeStatusTransition),
+		UpdateContext: provider.UpdateWithPooledClient(updateTaskManagementWorkTypeStatusTransition),
+		DeleteContext: provider.DeleteWithPooledClient(deleteTaskManagementWorkTypeStatusTransition),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"worktype_id": {
+				Description: `The id of the worktype this status belongs to. Changing this attribute will cause the status to be dropped and recreated.`,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+			},
+			`status_id`: {
+				Description:  `Name of the status.`,
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.StringLenBetween(3, 256),
+			},
+			`destination_status_ids`: {
+				Description: `A list of destination Statuses where a Workitem with this Status can transition to. If the list is empty Workitems with this Status can transition to all other Statuses defined on the Worktype. A Status can have a maximum of 24 destinations.`,
+				Optional:    true,
+				Type:        schema.TypeList,
+				Elem: &schema.Schema{
+					Type:      schema.TypeString,
+					StateFunc: modifyStatusIdStateValue,
+				},
+				MaxItems: 24,
+			},
+			`default_destination_status_id`: {
+				Description: `Default destination status to which this Status will transition to if auto status transition enabled.`,
+				Optional:    true,
+				StateFunc:   modifyStatusIdStateValue,
+				Type:        schema.TypeString,
+			},
+			`status_transition_delay_seconds`: {
+				Description:  `Delay in seconds for auto status transition. Required if default_destination_status_id is provided.`,
+				Optional:     true,
+				Type:         schema.TypeInt,
+				ValidateFunc: validation.IntAtLeast(60),
+			},
+			`status_transition_time`: {
+				Description: `Time is represented as an ISO-8601 string without a timezone. For example: HH:mm:ss`,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+		},
+	}
+}
+
+// TaskManagementWorktypeStatusTransitionExporter returns the resourceExporter object used to hold the genesyscloud_task_management_worktype_status exporter's config
+func TaskManagementWorktypeStatusTransitionExporter() *resourceExporter.ResourceExporter {
+	return &resourceExporter.ResourceExporter{
+		GetResourcesFunc: provider.GetAllWithPooledClient(getAllAuthTaskManagementWorkTypeStatusTransition),
+		RefAttrs: map[string]*resourceExporter.RefAttrSettings{
+			"worktype_id":                   {RefType: "genesyscloud_task_management_worktype"},
+			"destination_status_ids.*":      {RefType: "genesyscloud_task_management_worktype_status"},
+			"default_destination_status_id": {RefType: "genesyscloud_task_management_worktype_status"},
+		},
+	}
+}
```

Comment:
Is there any need for this exporter? It seems like the same worktype statuses could be exported as both "genesyscloud_task_management_worktype_status" and "genesyscloud_task_management_worktype_status_transition" if the user includes both resource types in the export